test/cli/migrate-make.spec.js

```diff
@@ -213,4 +259,94 @@ development: {
     );
     expect(fileCount).to.equal(1);
   });
+
+  it('Create a new migration using --stub </file/path> relative to the knexfile', async () => {
```

Comment:
I would extract all of them into a separate `describe()` block; probably need to add one for existing ones as well then.